osu.Game.Tests/Chat/MessageNotifierTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using NUnit.Framework;
+using osu.Game.Online.Chat;
+
+namespace osu.Game.Tests.Chat
+{
+    [TestFixture]
+    public class MessageNotifierTests
+    {
+        private readonly MessageNotifier messageNotifier = new MessageNotifier();
+
+        [Test]
+        public void TestMentions()
+        {
+            // Message (with mention, different casing)
+            Assert.IsTrue(messageNotifier.IsMentioning("Hey, Somebody Playing OSU!", "Somebody playing osu!"));
```

Comment:
these tests are really strange. you should be testing by passing in `Message` objects, not strings.